src/olympia/abuse/models.py

```diff
@@ -159,8 +160,40 @@ def get_cinder_reporter(cls, abuse):
             )
         return reporter
 
+    @classmethod
+    def handle_already_removed(cls, abuse_report):
+        decision = CinderDecision(
+            addon=abuse_report.addon,
+            rating=abuse_report.rating,
+            collection=abuse_report.collection,
+            user=abuse_report.user,
+            action=DECISION_ACTIONS.AMO_CLOSED_NO_ACTION,
+        )
+        entity_helper = cls.get_entity_helper(
+            abuse_report.target,
+            addon_version_string=abuse_report.addon_version,
+            resolved_in_reviewer_tools=False,
+        )
+        decision.id = entity_helper.create_decision(
+            action=decision.action.api_value,
+            reasoning='',
+            policy_uuids=(),
+        )
```

Comment:
This is likely the cause of the error I'm seeing locally - it should be `decision.cinder_id`, right ?

Reply:
ugh, last minute refactoring strikes away.  (Though it's weird/disappointing the CI didn't catch it)

Reply:
https://github.com/mozilla/addons-server/pull/22521/commits/c65b66d0e605b31859651e43342f0c676bedeeaf for the fix.  And I made an [attempt](https://github.com/mozilla/addons-server/pull/22521/commits/f4ebf7985ac6d8e439192f5f8eb716c1f7fe7a48) to cover other places we were returning a non-uuid style string as a uuid in the create_decision response.